package.json

```diff
@@ -55,7 +55,7 @@
     "dev-agones-silent": "npm run dev-agones &> /dev/null",
     "dev-client": "cd packages/client && npm run dev",
     "dev-taskserver": "cd packages/taskserver && npm run dev",
-    "dev-docker": "cd scripts && ./start-containers.sh",
+    "dev-docker": "cd scripts && ./start-containers.sh && sleep 10",
```

Comment:
this is sketch but it works. We launch our app too quickly and sql usually isn't up yet.